src/yarpmotorgui/main.cpp

```diff
@@ -158,6 +157,12 @@ int main(int argc, char *argv[])
                     }
                 }
             }
+
+            if (desc_driver)
+            {
+                desc_driver->close();
+                delete desc_driver;
+            }
```

Comment:
`desc_driver=nullptr` ?

Reply:
Not really necessary, since the pointer goes out of scope right after the delete...